### PR TITLE
Updated the class name of TranslationTextComponent

### PR DESCRIPTION
**Pull request by GeldornOfDragonhill**
Just a name update for 1.15

### CODE REVIEW

File: docs/concepts/internationalization.md [hunk 1]
```diff
@@ -43,22 +43,22 @@ Localization methods
...
+The first parameter of the `TranslationTextComponent(String, Object...)` constructor is a translation key, and the rest are used for formatting. The only supported format specifiers are `%s` and `%1$s`, `%2$s`, `%3$s` etc. Formatting arguments may be other `ITextComponent`s that will be inserted into the resulting formatted text with all their attributes preserved.
 
 ### `TextComponentHelper`
 
-- `createComponentTranslation(ICommandSender, String, Object...)` creates a localized and formatted `ITextComponent` depending on a receiver. The localization and formatting is done eagerly if the receiver is a vanilla client. If not, the localization and formatting is done lazily with a `TextComponentTranslation`. This is only useful if the server should allow vanilla clients to connect.
+- `createComponentTranslation(ICommandSender, String, Object...)` creates a localized and formatted `ITextComponent` depending on a receiver. The localization and formatting is done eagerly if the receiver is a vanilla client. If not, the localization and formatting is done lazily with a `TranslationTextComponent`. This is only useful if the server should allow vanilla clients to connect.
```
Comment (Aeltumn) [inline docs/concepts/internationalization.md:62]: `ICommandSender` changed to `ICommandSource`

Comment (Aeltumn) [inline docs/concepts/internationalization.md:62]: The method returns a `TextComponent`, not a `ITextComponent`

**Review by Aeltumn — APPROVED** [review]

**Comment by Aeltumn** [comment]
I forgot to add a comment to my review the other day. I think this PR is fine to merge in but you could update the other names too.

**Review by tterrag1098 — CHANGES_REQUESTED** [review]
Please be thorough and update all the class names in this area.

**Comment by tterrag1098** [comment]
Also, this should target 1.14.x if it's relevant there.

**Comment by sciwhiz12** [comment]
This PR has been superseded by #331; closing.

**Pull request closed without merging**